config/jobs/kubeflow/kubeflow-periodics.yaml

```diff
@@ -182,6 +182,38 @@ periodics:
   annotations:
     testgrid-dashboards: sig-big-data
     description: Periodic testing of Kubeflow examples
+  #******************************************************************************
+  # kubeflow/pipelines
+  #******************************************************************************
+- name: kubeflow-pipelines-periodic-functional-test
+  cluster: kubeflow
+  interval: 20m
```

Comment:
suggest 5m, because the test is very lightweight

Reply:
done